src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -70,10 +72,18 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     dir
   }
   private val draftsDir = getConfiguredDirectory("deposits.drafts")
-  private lazy val submitter = new Submitter(
-    stagingBaseDir = getConfiguredDirectory("deposits.stage"),
-    submitToBaseDir = getConfiguredDirectory("deposits.submit-to")
-  )
+  private val stagingDir = getConfiguredDirectory("deposits.stage")
+  private val submitToDir = getConfiguredDirectory("deposits.submit-to")
+
+  private val group = configuration.properties.getString("deposit.permissions.group")
```

Comment:
could you do an extract method of these two variables into something like `private val principal = getPrinciple("deposit.permissions.group")` to have it more look like the `getConfiguredDirectory` calls above?